ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml

```diff
@@ -1024,26 +1024,32 @@ tests:
       BASE_DOMAIN: origin-ci-int-aws.dev.rhcloud.com
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-shared-vpc-phz
-- as: e2e-agent-compact-ipv4
+- as: e2e-agent-compact-ipv4-conformance
   cluster: build05
   cron: 0 */12 * * *
   steps:
     cluster_profile: equinix-ocp-metal
-    workflow: agent-e2e-compact-ipv4
+    env:
+      DEVSCRIPTS_CONFIG: "AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4  \nFIPS_MODE=true\n"
+    workflow: agent-e2e-generic
   timeout: 5h0m0s
-- as: e2e-agent-sno-ipv6
+- as: e2e-agent-sno-ipv6-conformance
   cluster: build05
   cron: 0 */12 * * *
   steps:
     cluster_profile: equinix-ocp-metal
-    workflow: agent-e2e-sno-ipv6
+    env:
+      DEVSCRIPTS_CONFIG: "AGENT_E2E_TEST_SCENARIO=SNO_IPV6  \nFIPS_MODE=true\n"
```

Comment:
For the presubmits ipv6/ha we didn't yet enabled the fips mode, maybe it's worth trying it on the presubmits first?